internal/platform/auth/midware.go

```diff
@@ -32,6 +39,14 @@ func Midware(publicKeyBase64Str string) (app.Middleware, error) {
 			// Extract the token from the Authorization header provided on the request.
 			tokenString := c.Request.Header.Get("Authorization")
 
+			if tokenString == "" && config.AllowQueryString {
+
+				// In the event that the request does not have a header key for the
+				// Authorization header, then we need to try and access it from the URL
+				// query parameters if this was enabled from our config.
+				tokenString = c.Request.URL.Query().Get("access_token")
+			}
+
 			if tokenString == "" {
```

Comment:
What if the tokenString is empty and config.AllowQueryString is false in the above statement. Then this if will always be false? I feel like something is off between these two if statements.